content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md

```diff
@@ -397,15 +397,15 @@ Type `str` accepts string literals `"string value"`.
 Type `bytes` accepts string literals with base64 values. For example to set binary array `{0, 1, 2, 3}` we should 
 encode it to base64: `"AAECAw=="`. AcraServer will decode base64 values and pass it as binary array according to database's protocol.
 
-#### **tokenized**
+#### **tokenized** (deprecated since 0.93.0)
 
 Required: `false`
 
 Type: `boolean`
 
 Group: `tokenization`
 
-Description: turns on tokenization for column.
+Description: turns on tokenization for column. Deprecated since 0.93.0 - use non-empty `token_type` instead.
```

Comment:
```suggestion
Description: turns on tokenization for column. Since Acra 0.93.0 this flag is deprecated. It's enough to specify `token_type` value for a column to enable tokenization for that column with specified data type.
```